tests/full_test.py

```diff
@@ -0,0 +1,40 @@
+from utils.MeasurementExporter import MeasurementExport
+
+def test_main():
+
+    import os
+    import flywheel
+
+    fw = flywheel.Client(os.environ["FWGA_API"])
+    # Get configuration, acquisition, and file info
+    from pathlib import Path
+
+    parent_acq = "621d449c559d4f2a0d1468e0"
+    file_id = "621d449f332f06de9645fd7a"
+    file_obj = fw.get_file(file_id)
+    input_file = "/Users/davidparker/Documents/Flywheel/SSE/MyWork/Gears/roi2nix/tests/test_ax_cor_sag/Scans/T1_SE_AX.zip"
```

Comment:
Can you make these assets part of the gear and not use an absolute path?

Reply:
NO! MINE!